indico/htdocs/sass/modules/_surveys.scss

```diff
@@ -28,6 +28,17 @@
     padding: 1em;
 }
 
+%survey-questionnaire-handle {
+    @include icon-before('icon-grid2');
+    color: lighten($gray, 10%);
```

Comment:
Doesn't any of the colors in the palette fits? Not even applying `$color-variation` instead of arbitrary `10%`?